graphs/0035_GraphNode_Basic.json

```diff
@@ -30,7 +30,7 @@
       "component": "rdf-components/RequestTemplate",
       "metadata": {
         "label": "LOAD INTO GRAPH",
-        "x": 324,
+        "x": 288,
```

Comment:
I think there should be something here to document/demonstrate how to use a request template, but perhaps a more generic example would be better.